app/helpers/share_helper.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ShareHelper
+  def share_image_url
+    return request.host unless defined?(event)
+
+    params = {
+      :s => image_url("/images/onruby_share.svg"),
+      :title => page_title,
+      :subtitle => "#{I18n.t("show.hosted_by")} #{event.user.name}",
+      :date => "#{I18n.l(event.date, format: :long)}",
+      :location => event.location ? "#{I18n.t('show.at')} #{event.location.name}, #{event.location.address}" : "",
+      :participants => event.participants.count > 1 ? event.participants.count : " ",
+      :logo => image_url("labels/#{event.label}.png")
+    }
+
+    return "https://p.salmo.link/render.png?#{params.to_query}"
```

Comment:
```suggestion
    "https://p.salmo.link/render.png?#{params.to_query}"
```

was ist denn das für ein service?

Reply:
habsch gebaut. https://github.com/p0wl/randarar

Reply:
harrr 🚀